backend/apis/leads/get-lead.api.js

```diff
@@ -97,7 +97,7 @@ function getLeadById(leadId, cbk, connection) {
           ON leadEvents.leadId = leads.id
         INNER JOIN events
           ON events.id = leadEvents.eventId
-      WHERE leads.id = ? AND leads.isDeleted = false;
+      WHERE leads.id IN (?) AND leads.isDeleted = false;
```

Comment:
Why did this change? You're allowing for leadId to be an array of leads now? But we're still coercing it to a number on line 41.